code/modules/nano/interaction/default.dm

```diff
@@ -102,3 +102,10 @@
 		. = min(., shared_living_nano_distance(src_object))
 		if(. == STATUS_UPDATE && (TK in mutations))	// If we have telekinesis and remain close enough, allow interaction.
 			return STATUS_INTERACTIVE
+
+/mob/living/carbon/alien/default_can_use_topic(var/src_object)
+	. = shared_nano_interaction(src_object)
+	if(. != STATUS_CLOSE)
+		. = min(., shared_living_nano_distance(src_object))
+		if(!IsAdvancedToolUser())
+			. = STATUS_UPDATE
```

Comment:
They can't even look at computer screens if they don't have advanced tool user status; They just smash the console.